addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/virtual/StateContactVirtualDatapointHandler.java

```diff
@@ -71,12 +71,13 @@ private boolean isApplicable(HmDevice device) {
     }
 
     private Boolean convertState(Object value) {
-        if (value == null) {
+        if (value == null || !(value instanceof Integer)) {
             return null;
         }
-        if ("CLOSED".equals(value)) {
+        int intValue = (int) value;
```

Comment:
This conversion is not needed, you checked it to be an instance of Integer and https://stackoverflow.com/a/30454826/3656774